tensorflow/docs_src/programmers_guide/saved_model.md

```diff
@@ -4,7 +4,7 @@ This document explains how to save and restore
 @{$variables$variables} and models.
 
 Important: TensorFlow model files are code. Be careful with untrusted code.
-See [Using TensorFlow Securely](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/SECURITY.md)
+See [Using TensorFlow Security](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/SECURITY.md)
```

Comment:
The document is about using TensorFlow in a secure manner, so Using TensorFlow Securely is correct.